sass/_component_versionpicker_fix.scss

```diff
@@ -8,13 +8,14 @@ These overrides are necessary to make the version picker work with the Sphinx Wa
   bottom: 50px;
   right: 50px;
   max-width: $modal-sm;
-  max-height: calc(100vh - 50px);
+  max-height: calc(100vh 50px);
   overflow-y: auto;
   background: #1f1d1d;
   z-index: 1050;
 
   &.shift-up .rst-other-versions {
     display: block;
+    height: 80vh;
```

Comment:
This is not necessary. Instead of adding this, this block should be broken up to separate the `.shift-up` and `.rst-other-versions`, and the `max-height: calc(100vh 50px);` should be moved to inside just the `.shift-up` selector, e.g.

```scss
  &.shift-up {
    max-height: calc(100vh - 50px);

    &.rst-other-versions {
      display: block;
    }
  }
```

Reply:
Sorry It was a little  a little typo and I missed `-`  implementing the changes you proposed.